spec/git.spec.js

```diff
@@ -24,11 +24,13 @@ describe('Git', function () {
       var version = /v([0-9]+)/.exec(process.version)[ 1 ]
       var today = new Date()
       var lts = git.checkLTS()
-      if (version === '6' && today < new Date('10-01-2017')) {
```

Comment:
Let's just do `new Date(2017, 9, 1)` so we aren't fighting localization settings.

Reply:
Assuming I'm in UTC+7 there is a differences in these three options. I'd keep @owenniblock's choice. 

```js
> new Date('2017-01-10')
2017-01-10T00:00:00.000Z

> new Date('10-01-2017')
2017-09-30T17:00:00.000Z

> new Date(2017, 9, 1)
2017-09-30T17:00:00.000Z
```

**But in general I'd rather prefer to use ISO strings everywhere.**
```js
new Date('2017-10-01T00:00:00.000Z')
```

Reply:
However, I think ISO stuff is out of scope of this PR. I'm good to keep as is.